fuzzyfinder.go

```diff
@@ -81,6 +87,8 @@ func (f *finder) initFinder(items []string, matched []matching.Matched, opts []O
 	}
 	f.opt = &opt
 
+	f.state.clear()
```

Comment:
I designed `state` struct to make good use of zero-values. So, it can be initialized with just empty `state` instance.

``` go
f.state = state{}
```

Reply:
Makes sense 👍